Branch-SDK/src/io/branch/referral/BranchStrongMatchHelper.java

```diff
@@ -164,7 +164,7 @@ private Uri buildStrongMatchUrl(String matchDomain, DeviceInfo deviceInfo, PrefH
             String hardwareIDTypeVal = deviceInfo.isHardwareIDReal() ? Defines.Jsonkey.HardwareIDTypeVendor.getKey() : Defines.Jsonkey.HardwareIDTypeRandom.getKey();
             uriString += "&" + Defines.Jsonkey.HardwareIDType.getKey() + "=" + hardwareIDTypeVal;
             // Add GAID if available
-            if (systemObserver.GAIDString_ != null && !BranchUtil.isTestModeEnabled(context)) {
+            if (systemObserver.GAIDString_ != null && !BranchUtil.isTestModeEnabled()) {
```

Comment:
Does this mean we send the GAID when `isDebugModeEnabled`? If so, I think this is a deviation

Reply:
Outdated.  Now using `BranchUtil.checkTestMode()` to determine if the SDK is in test mode or not.